extension/src/prototypes/view/ui-containers/DatasetTypeList.tsx

```diff
@@ -145,9 +145,9 @@ export const DatasetTypeList: FC = () => {
       expanded={expanded}
       onNodeToggle={handleNodeToggle}
       maxheight={gridHeight - searchHeaderHeight}>
-      {filteredDatasetTypeOrder?.map(datasetType => (
+      {filteredDatasetTypeOrder?.map((datasetType, index) => (
         <DatasetTypeItem
-          key={datasetType.code}
+          key={index} // datasetType.code is not unique in this list
```

Comment:
Please help check whether it's okey. 
Basically it's not related with the task of this PR, i just notice there's react warnings like list item has the same key `tran` here.